docker/docker-compose.mariadb.yml

```diff
@@ -0,0 +1,23 @@
+# Override to use MariaDB as a backing store for datahub-gms.
```

Comment:
Maybe move these to their respective directory under `docker/<db type>` to declutter the root?

Reply:
Done